mro/__init__.py

```diff
@@ -35,96 +35,120 @@ def execute_sql(sql, values=None):
 
 
 def _load_standard_db(connection):
```

Comment:
Added print statements to make it easier to track where the time is going (at a glance) when viewing pod logs